todo_app/data/session_items.py

```diff
@@ -1,84 +1,93 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
-
-def get_items():
+from logging import Formatter
+import os
+import requests
+import arrow
+from pprint import pp, pprint
+from dotenv import load_dotenv
+
+load_dotenv()
+
+class List:  
+    def __init__(self, id, name):  
+        self.id = id  
+        self.name = name
+
+class Card:  
+    def __init__(self, id, name, idList, desc, due):  
+        self.id = id  
+        self.name = name
+        self.idList = idList
+        self.desc = desc
+        if due:
+            self.due = arrow.get(due).datetime.strftime('%d %B %Y')
+        else:
+            self.due = ''
+
+def get_lists():
     """
-    Fetches all saved items from the session.
+    Fetches all Lists from the given Trello Board.
 
     Returns:
-        list: The list of saved items.
+        JSON response.
     """
-    return session.get('items', _DEFAULT_ITEMS)
+    lists = []
 
+    for list in (requests.get("https://api.trello.com/1/boards/{}/lists?key={}&token={}".format(os.environ.get('BOARD_ID'),os.environ.get('SECRET_KEY'),os.environ.get('SECRET_TOKEN')))).json():
```

Comment:
A format string literal is a bit easier to read
```suggestion
    for list in (requests.get(f"https://api.trello.com/1/boards/{os.environ.get('BOARD_ID')}/lists?key={os.environ.get('SECRET_KEY')}&token={os.environ.get('SECRET_TOKEN')}").json():
```

Note the `f` before the quotes.

Also you could look at using the `params` optional paramter of requests.get instead of building the query string yourself. It would make commonising it a bit easier too, e.g. having a get_auth_params() function that returns `{ 'key': os.environ.get('SECRET_KEY', 'token': etc }` which all of your requests can use.